feature-libs/storefinder/components/store-finder-list-item/store-finder-list-item.component.html

```diff
@@ -50,7 +50,7 @@
       href="{{ getDirections(location) }}"
       target="_blank"
       rel="noopener noreferrer"
-      class="btn btn-sm btn-secondary btn-block cx-button"
```

Comment:
A small change, but I think it's still considered breaking. Let's use a feature flag here.